docs/Spatial.md

```diff
@@ -1,6 +1,92 @@
+|[__phidget__.quit](https://github.com/RIAEvangelist/node-phidget-API/blob/master/docs/Phidget.md)|N/A |This method requests a disconnect from the phidget board.  The disconnected event will be dispatched when the connection has been successfully disconnected. |
+|observeGyro|gyro|Used for asynchronously observing the changes to the PhidgetSpatial board's gyroscopic sensors. This behaves much like the JS [Object.observe](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/observe), however you only pass the handler, not the object or accept list. All of this is handled by the Phidget Framework.|
+|observeAcceleration|accel|Used for asynchronously observing the changes to the PhidgetSpatial board's acceleration axes. This behaves much like the JS [Object.observe](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/observe), however you only pass the handler, not the object or accept list. All of this is handled by the Phidget Framework. |
+|observeMagneticField|mag|Used for asynchronously observing the changes to the PhidgetSpatial board's magnetic field sensors. This behaves much like the JS [Object.observe](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/observe), however you only pass the handler, not the object or accept list. All of this is handled by the Phidget Framework. |
+|zeroGyro|N/A| Re-zeroes the gyroscope.|
+|setCompassCorrectionParameters|`magfield, offset0, offset1, offset2, gain0, gain1, gain2, T0, T1, T2, T3, T4, T5`|This function adjusts the parameters of the compass.|
+|resetCompassCorrectionParameters|N/A|Resets the Compass Correction Parameters to default values:`(1,0,0,0,1,1,1,0,0,0,0,0,0)`|
+
+##Data
+ 
+|Key|Data Type|Writable|Description|
+|---|---|---|---|
+|type|string|no|'PhidgetSpatial'|
+|accelerationMin|number|no|Minimum acceleration.|
+|accelerationMax|number|no|Maximum acceleration.|
+|angularRateMax|number|no|Minimum angular velocity.|
+|angularRateMin|number|no|Maximum angular velocity.|
+|dataRate|number|yes|The number of times data is acquired per second. |
+|dataRateMax|number|no|Maximum data rate in samples per second.|
+|dataRateMin|number|no|Minimum data rate in samples per second.|
+|compassAxisCount|number|no|Number of magnetic field sensors on your board.|
+|gyroAxisCount|number|no|Number of gyroscopic sensors on your board.|
+|accelerationAxisCount|number|no|Number of acceleration sensors on your board.|
+|magneticFieldMax|number|no|Maximum magnetic field in G (Gauss).|
+|magneticFieldMin|number|no|Minimum magnetic field in G (Gauss).|
+|acceleration|array|no|Contains three directions of acceleration in g.|
+|gyro|array|no|Contains three directions of angular velocity degrees per second.|
+|magneticField|array|no|Contains three directions of magnetic field in G (Gauss).|
```

Comment:
Probably don't need contains for Arrays, should have used Gauss instead of G (Gauss). This looks like a way to prevent confusion between G and g. Generally speaking, if you have to clarify with parentheses you probably should use the clear version instead of the version you are clarifying to avoid stating the same information twice.